src/sql/src/plan/statement/dml.rs

```diff
@@ -983,15 +985,14 @@ fn plan_copy_to(
         _ => sql_bail!("only AWS CONNECTION is supported for COPY ... TO <expr>"),
     }
 
-    if format != CopyFormat::Csv {
-        sql_bail!("only CSV format is supported for COPY ... TO <expr>");
-    }
-
-    // TODO(mouli): Get these from sql options
-    let format_params = CopyFormatParams::Csv(
-        CopyCsvFormatParams::try_new(None, None, None, None, None)
-            .map_err(|e| sql_err!("{}", e))?,
-    );
+    let format = match format {
+        CopyFormat::Csv => S3SinkFormat::PgCopy(CopyFormatParams::Csv(
+            // TODO(mouli): Get these from sql options
+            CopyCsvFormatParams::try_new(None, None, None, None, None)
+                .map_err(|e| sql_err!("{}", e))?,
+        )),
+        _ => sql_bail!("only CSV format is supported for COPY ... TO <expr>"),
```

Comment:
it would be more accurate to `panic!` here, but your choice!

Reply:
ah this is changed in a later commit, but switched to `bail_unsupported!`. I think we want to tell the user that they format they tried is unsupported rather than panicking, right?

Reply:
Its just that its checked in a previous if statement! You can leave it like this!